Cloud Application Manager/Core Concepts/boxes.md

```diff
@@ -18,56 +20,73 @@
+* [Deployment Policy](../Automating Deployments/deploymentpolicy-box.md) - Select and share infrastructure resources, networking, and more from a cloud provider.
+
+Fill mandatory fields (with an *) and define some basic metadata described below.
+#### Box metadata
 
 | Metadata | Box Type | What it Means |
 |--------------|--------------|---------------------|
-| Requirements | Script,<br> Template,<br> Container | It’s good practice to tag the runtime that the box requires to deploy. Cloud Application Manager auto suggests tags like Linux, Ubuntu, Java and so on. When ready to launch the box, you are presented with deployment policies that match the requirements. These deployment policies provide the right infrastructure or services the box needs to deploy.<br>**Note:** In CloudFormation boxes, the tags help look instances for bindings. |
-| Automatic<br>Updates | Script,<br>Template,<br>Container | Select the level of updates to automatically apply to instances you launch of a box version:<br>+ **Off** - It’s turned off by default.<br>+ **All Updates** - Applies all changes.<br>+ **Minor & Patch Updates** - Applies minor and patch changes to the version deployed.<br>+ **Patch Updates** - Applies only the patch changes to the version deployed. |
-| Provider | Deployment<br>Policy | Select the cloud provider registered in Cloud Application Manager for which you will carve out infrastructrure resources in the policy. |
-| Claims | Deployment<br>Policy | Tag the services and infrastructure that a policy provides like Linux, Ubuntu 12.04, load balancing, and so on for deployments. Add claims so that boxes with matching requirements can successfully deploy using the right policy. |
+| Name,<br/>Description,<br/>Icon | All | Give it a name and optionally a description and an icon. |
+| Requirements | Script,<br/> Template,<br/> Container | It’s good practice to tag the runtime that the box requires to deploy. Cloud Application Manager auto suggests tags like Linux, Ubuntu, Java and so on. When ready to launch the box, you are presented with deployment policies that match the requirements. These deployment policies provide the right infrastructure or services the box needs to deploy.<br/>**Note:** In CloudFormation boxes, the tags help you to look for binding instances. |
+| Automatic<br/>Updates | Script,<br/>Template,<br/>Container | Select the level of updates to automatically apply to instances you launch of a box version:<br/>+ **Off** - It’s turned off by default.<br/>+ **All Updates** - Applies all changes.<br/>+ **Minor & Patch Updates** - Applies minor and patch changes to the version deployed.<br/>+ **Patch Updates** - Applies only the patch changes to the version deployed. |
+| Provider | Deployment<br/>Policy | Select the cloud provider registered in Cloud Application Manager for which you will carve out infrastructrure resources in the policy. |
```

Comment:
This is only for deployment policy box types, so I would only explain here common fields. Please remove this and any other specific one, and ensure they are explained on each box type article/section

Reply:
Box type column specifies where these fields are used. Finally, this page is referenced from many links to display just that information that's why they are keeping in this table.